src/features/currentTodo.ts

```diff
@@ -1,10 +1,16 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { PayloadAction, Slice, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
 
-export const currentTodoSlice = createSlice({
+export const currentTodoSlice: Slice<Todo | null> = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    setCurrentTodo(_, action: PayloadAction<Todo | null>) {
+      return action.payload;
+    },
```

Comment:
In my view, strange thing. 
The reducer needs to change state. 
But in this case, we do not do that. Why do we need them?